core/spec/models/spree/order/payment_spec.rb

```diff
@@ -69,10 +69,6 @@ module Spree
 
     context "ensure source attributes stick around" do
       # For the reason of this test, please see spree/spree_gateway#132
```

Comment:
I dont think we should, the test that is in this context is the regression for the issue mentioned in the comment.